packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/index.js

```diff
@@ -77,19 +72,9 @@ const EditView = () => {
       },
     });
 
-  const createWebhookMutation = useMutation((body) =>
-    request('/admin/webhooks', {
-      method: 'POST',
-      body,
-    })
-  );
+  const createWebhookMutation = useMutation((body) => post('/admin/webhooks', body));
 
-  const updateWebhookMutation = useMutation(({ id, body }) =>
-    request(`/admin/webhooks/${id}`, {
-      method: 'PUT',
-      body,
-    })
-  );
+  const updateWebhookMutation = useMutation(({ id, body }) => post(`/admin/webhooks/${id}`, body));
```

Comment:
Both need to be wrapped in try / catch.

Reply:
These have error handling when the mutation is called